91-grub-mkconfig.install

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# this file is installed by sys-kernel/installkernel-gentoo
+
+set -euo pipefail
+IFS=$'\n\t'
+
+: "${GRUB_CFG:=/boot/grub/grub.cfg}"
+
+# familliar helpers, we intentionally don't use gentoo functions.sh
+die() {
+	echo -e "${NOCOLOR-\e[1;31m*\e[0m }${*}" >&2
+	exit 1
+}
+
+einfo() {
+        echo -e "${NOCOLOR-\e[1;32m*\e[0m }${*}"
+        return 0
+}
+
+main() {
+	# re-define for subst to work
+	[[ -n ${NOCOLOR+yes} ]] && NOCOLOR=
+
+	# do nothing if somehow grub is not installed
+	[[ -x $(command -v grub-mkconfig) ]] || exit 0
+
+	[[ ${EUID} -eq 0 ]] || die "Please run this script as root"
+
+	if [[ -f ${GRUB_CFG} ]]; then
+		einfo "Backing up existing grub config as ${GRUB_CFG}~"
+		cp -v "${GRUB_CFG}"{,~} || die "Failed to save existing config"
+	fi
+
+	einfo "Generating new grub config as ${GRUB_CFG}"
+	local dname
+	dname="$(dirname "${GRUB_CFG}")"
```

Comment:
```suggestion
	dname=${GRUB_CFG##*/}
```

Reply:
you mean `"${C%/*}"` ? thats dirname, not basename.

Reply:
Jes.